src/sp_config_keywords.c

```diff
@@ -229,17 +231,24 @@ int parse_disabled_functions(char *line) {
   MUTUALLY_EXCLUSIVE(df->r_function, df->function, "r_function", "function");
   MUTUALLY_EXCLUSIVE(df->filename, df->r_filename, "r_filename", "filename");
   MUTUALLY_EXCLUSIVE(df->ret, df->r_ret, "r_ret", "ret");
+  MUTUALLY_EXCLUSIVE(df->key, df->r_key, "r_key", "key");
```

Comment:
tbd : add a test to check that key/key_r are mutually exclusive with value/value_r